runtime/doc/api.txt

```diff
@@ -400,26 +442,26 @@ nvim_set_current_dir({dir})                           *nvim_set_current_dir()*
                     {dir}  Directory path
 
 nvim_get_current_line()                              *nvim_get_current_line()*
-                Gets the current line
+                Gets the current line.
```

Comment:
@neovim/core 	Even though it seems to be a doxygen(?) convention to omit the period in the first sentence, it does not make sense for us, because in contexts such as Vim help docs it looks like a mistake. 

Also it's inconsistent: we terminate almost every other docstring, even sentence fragments in parameter descriptions ...